workspaces/arborist/lib/shrinkwrap.js

```diff
@@ -10,7 +10,7 @@
 // definitely not before npm v8.
 
 const localeCompare = require('@isaacs/string-locale-compare')('en')
-const defaultLockfileVersion = 2
```

Comment:
How hard would it be to fix this?  I know we had planned on moving this into npm properly and not making arborist make this decision.